src/components/data-exploration/generic-gene-table/SelectionActions.jsx

```diff
@@ -71,9 +77,21 @@ const SelectionActions = (props) => {
     );
   };
   return (
-    <Space style={{ float: 'left' }}>
+    <Row style={{ float: 'left', paddingRight: '50px' }}>
+      {
+        showCSV ? (
```

Comment:
I see that the block under `showCSV` does not only contain CSV - it also contains the button to pathway analysis. Can we rename this conditional better?

Reply:
Seems like this property is generated in `GeneTable` from the `existence of `onExportCSV`. Time to separate the  variable from the value of `onCSVExport`? `GeneTable` is used in `GeneListTool` and `DiffExprResults`.

Reply:
after taking a look I moved both export csv and pathway analysis up to `DiffExprResults` (and passed through `extraOptions`), that way we don't have to worry about the ifs and the handlers going down like 4 levels when they are not even used in the other use case of these components.